src/services/item/plugins/geolocation/index.test.ts

```diff
@@ -493,8 +495,8 @@ describe('Item Geolocation', () => {
           method: HttpMethod.Get,
           url: `${ITEMS_ROUTE_PREFIX}/geolocation/reverse`,
           query: {
-            lat: 2,
-            lng: 2,
+            lat: '2',
+            lng: '2',
```

Comment:
You cannot use number here? Maybe it becomes worth to use json stringify... or build the url manually, not sure.

Reply:
@pyphilia This has nothing to do with our schema. This is because the app.inject options takes only strings as inputs for the query parameters. They will be validated and parsed and when needed cast to numbers as usual when received. So I think it is fine.